wsdl/ver10/schema/onvif.xsd

```diff
@@ -7631,6 +7631,121 @@ and sample rate. </xs:documentation>
 		<xs:anyAttribute processContents="lax"/>
 	</xs:complexType>
 	<!--===============================-->
+	<xs:simpleType name="TargetFormat">
+		<xs:restriction base="xs:string">
+			<xs:enumeration value="MP4">
+				<xs:annotation>
+					<xs:documentation>
+						MP4 files with all tracks in a single file.
+					</xs:documentation>
+				</xs:annotation>
+			</xs:enumeration>
+			<xs:enumeration value="CMAF">
+				<xs:annotation>
+					<xs:documentation>
+						CMAF compliant MP4 files with 1 track per file.
+					</xs:documentation>
+				</xs:annotation>
+			</xs:enumeration>
+		</xs:restriction>
+	</xs:simpleType>
+	<!--===============================-->
+	<xs:simpleType name="EncryptionMode">
+		<xs:restriction base="xs:string">
+			<xs:enumeration value="CENC">
```

Comment:
Add CBCS

Reply:
It was a point I was going to bring to tomorrow's meeting. It was unclear to me if CENC was simply the value as its the name of the specification or if it was specifically the 'cenc' mode from the ISO spec. Based on your comment, I am assuming it was meant to be the 'cenc' CRT scheme?

Reply:
I will add CBCS and clear this up. Indeed, CENC meant "AES-CTR mode full sample and video NAL Subsample encryption".